OpenLIFUTransducerTracker/OpenLIFUTransducerTracker.py

```diff
@@ -1456,11 +1457,12 @@ def onStartPhotoscanGenerationButtonClicked(self):
                     meshroom_pipeline = photoscan_generation_options_dialog.get_selected_meshroom_pipeline(),
                     image_width = photoscan_generation_options_dialog.get_entered_image_width(),
                 )
+                self.updatePhotoscanGeneratorProgressBar(new_photoscan_generator_progress_value = 100)
             except CalledProcessError as e:
                 slicer.util.errorDisplay("The underlying Meshroom process encountered an error.", "Meshroom error")
                 raise e
-            self.updatePhotoscanGeneratorProgressBar(new_photoscan_generator_progress_value = 100)
-            self.ui.photoscanGenerationStatusMessage.hide()
+            finally:
```

Comment:
Tested a case where it fails and a case where it succeeds -- indeed this worked, ty!